actionview/lib/action_view/template.rb

```diff
@@ -141,7 +141,7 @@ def initialize(source, identifier, handler, details)
       @compile_mutex     = Mutex.new
     end
 
-    # Returns if the underlying handler supports streaming. If so,
+    # Returns whether the underlying handler supports streaming. If so,
     # a streaming buffer *may* be passed when it start rendering.
```

Comment:
Since we're touching this anyway it would be great to expand the `*may*`. I don't understand what that means.